spiceypy/tests/test_wrapper.py

```diff
@@ -6648,16 +6732,44 @@ def test_trcnam():
     spice.chkout("test")
     assert spice.trcdep() == 0
     spice.reset()
+    do_test_trcoff_last('trcnam')
 
 
 def test_trcoff():
-    assert 1
+    do_test_trcoff_last('trcoff')
 
 
 def test_tsetyr():
-    spice.tsetyr(1969)
-    spice.tsetyr(2100)
-    spice.tsetyr(1969)
+    spice.reset()
+    # Expand 2-digit year to full year, typically 4-digit
+    tmp_getyr4 = lambda iy2: int(spice.etcal(spice.tparse('3/3/%02d' % (iy2,),22)[0]).split()[0])
+    # Find current lower bound on the 100 year interval of expansion,
+    # so it can be restored on exit
+    tsetyr_lowerbound = tmp_getyr4(0)
+    for iy2_test in xrange(100):
```

Comment:
oh and xrange doesn't exist in python 3, so I just use `range` for tests. If you really need the benefits of a generator in python 2.7 there is a python six function that I have used elsewhere in the code.